README.md

```diff
@@ -407,7 +409,7 @@ there, please let me know.
 
 For Mac users, at least the combination of macOS 11.2 and iTerm2 3.4.3 works.
 
-### Build and Install
+### Linux - Build and Install 
```

Comment:
I'd call this _Debian based systems_, as this is mostly targeting systems that are using the `apt` package manager, but it is independent e.g. from GNU/Linux or GNU/Hurd